docs/openhabian.md

```diff
@@ -17,52 +17,52 @@ However, setting up a fully working Linux system with all recommended packages a
 openHABian aims to provide a **self-configuring** Linux system setup specific to the needs of every openHAB user.
 To that end, the project provides two things:
 
-* Complete **SD-card images pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations for the Raspberry Pi and the Pine A64
-* The openHABian Configuration Tool to set up and configure openHAB and many related things on any Debian/Ubuntu based system
+*   Complete **SD-card images pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations for the Raspberry Pi and the Pine A64
```

Comment:
The people to do the openHAB docs have a guideline of "one sentence per line".
As some of our docs are forwarded there, we should stick to that guideline, too, at least on those documents
(docs/*) . (maybe put that in CONTRIBUTING.md, too)

Reply:
I didn't really touch the formatting of the docs/* files but I can add it in CONTRIBUTING.md

Reply:
ok